frontend/src/metabase/ui/components/theme/ThemeProvider/ThemeProvider.tsx

```diff
@@ -73,6 +75,7 @@ export const ThemeProvider = (props: ThemeProviderProps) => {
       theme={theme}
       getStyleNonce={() => window.MetabaseNonce ?? "metabase"}
       classNamesPrefix="mb-mantine"
+      cssVariablesSelector={isEmbeddingSdk ? ".mb-wrapper" : undefined}
```

Comment:
We use `ThemeProvider` in unexpected places such as `use-body-cell-measure.tsx` and `use-measure-column-width.tsx`, and that actually caused the styles to be injected to `:root` instead of the `cssVariablesSelector`. Let's add the `isEmbeddingSdk` condition here as it's very easy to forgot to add it in `mantineProviderProps` if someone else uses it in the codebase.